packages/renderer/src/lib/ui/DropdownMenu.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+import { faEllipsisVertical } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa/src/fa.svelte';
+export let backgroundColor = 'bg-zinc-800';
+export let enabled: boolean = true;
+
+// Show and hide the menu using clickOutside
+let showMenu = false;
+
+// If we touch outside the window, hide the menu
+let outsideWindow;
+
+// If we hit ESC while the menu is open, close it
+function handleEscape({ key }) {
+  if (key === 'Escape') {
+    showMenu = false;
+  }
+}
+
+function toggleMenu() {
+  showMenu = !showMenu;
+}
+
+// If we click outside the menu, close the menu
+function onWindowClick(e) {
+  if (outsideWindow.contains(e.target) == false) showMenu = false;
+}
+</script>
+
+<!-- Required in order for Svelte to track all key presses and if you pressed "ESC" -->
+<svelte:window on:keyup="{handleEscape}" on:click="{onWindowClick}" />
+
+<!-- If enabled, we'll show it, otherwise, just output the buttons normally.-->
+{#if enabled}
+  <!-- Create a "kebab" menu for additional actions. -->
+  <div class="relative inline-block text-left" bind:this="{outsideWindow}">
+    <!-- Button for the dropdown menu -->
+    <button
+      on:click="{() => {
+        toggleMenu();
+      }}"
+      class="mx-1 px-3 py-2 hover:text-violet-600 font-medium rounded-lg text-sm inline-flex items-center text-center {backgroundColor}">
+      <Fa class="h-4 w-4 text-xl" icon="{faEllipsisVertical}" />
+    </button>
+
+    <!-- Dropdown menu for all other actions -->
+    {#if showMenu}
+      <div
+        id="dropdown"
+        class="origin-top-right absolute right-0 z-10 m-2 rounded-md shadow-lg bg-zinc-700 ring-1 ring-black ring-opacity-5 divide-y divide-gray-600 focus:outline-none">
```

Comment:
`divide-violet-600` ?

it looks like also that the dark theme is not following the same state than on the navbar or on the container list
display: dark item, hovering = lighter

here we have a light gray 


I noticed a small issue: rounded corner seems to be off when hovering the element

https://user-images.githubusercontent.com/436777/202123795-62bf7a9f-f080-4b53-9a90-219b4038fa73.mp4